pkg/providers/ovirt/templates/os-mapper-provider.go

```diff
@@ -0,0 +1,61 @@
+package templates
+
+import (
+	"context"
+
+	"github.com/operator-framework/operator-sdk/pkg/k8sutil"
+	yaml "gopkg.in/yaml.v2"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	osConfigMapName    = "vmimport-os-mapper"
+	guestOsToCommonKey = "guestos2common"
+	osInfoToCommonKey  = "osinfo2common"
+)
+
+// OSMaps is responsible for getting the operating systems maps which contain mapping of GuestOS to common templates
+// and mapping of osinfo to common templates
+type OSMaps struct {
+	Client client.Client
```

Comment:
Is it likely that templates will be used be used by future providers? 
This looks to be provider-agnostic; what do you think about moving it to pkg/templates?

Reply:
:+1: